internal/authorizedapp/model/authorized_app_model.go

```diff
@@ -67,6 +68,14 @@ func (c *AuthorizedApp) IsAndroid() bool {
 	return c.Platform == androidDevice
 }
 
+func (c *AuthorizedApp) RegionsOnePerLine() string {
+	regions := []string{}
```

Comment:
Why not just `strings.Join(c.AllowedRegions, "\n")`?

Reply:
AllowedRegions is a map.

Reply:
c.AllowedRegions is a map

Reply:
well yup. you're right